moped-api/requirements/development.txt

```diff
@@ -1,84 +1,10 @@
-argcomplete==1.12.0
-attrs==20.2.0
-boto==2.49.0
-boto3==1.17.100
-botocore==1.20.100
-Cerberus==1.3.2
-certifi==2022.12.7
-cffi==1.14.3
-cfn-flip==1.2.3
-chardet==4.0.0
-cloudpickle==2.0.0
-cognitojwt==1.2.2
-croniter==1.0.15
-cryptography==39.0.1
-dask==2021.11.1
-distributed==2021.11.1
-docker==5.0.3
-durationpy==0.5
-ecdsa==0.14.1
-Flask==2.2.3
-Flask-Cognito==1.16
-Flask-Cors==3.0.9
-fsspec==2021.11.0
-future==0.18.3
-HeapDict==1.0.1
-hjson==3.0.2
-idna==2.10
-iniconfig==1.0.1
-itsdangerous==2.0.1
-Jinja2==3.0.3
-jmespath==0.10.0
-kappa==0.6.0
-locket==0.2.1
-MarkupSafe==2.1.1
-marshmallow==3.14.1
-marshmallow-oneofschema==3.0.1
-mock==4.0.2
-more-itertools==8.5.0
-moto==4.1.5
-msgpack==1.0.2
-mypy-extensions==0.4.3
-packaging==20.4
-partd==1.2.0
-pendulum==2.1.2
-pip-tools==5.3.1
-placebo==0.9.0
-pluggy==0.13.1
-prefect==0.15.9
-psutil==5.8.0
-py==1.10.0
-pyasn1==0.4.8
-pycparser==2.20
-pyparsing==2.4.7
-pytest==6.0.2
-pytest-mock-server==0.2.0
-python-box==5.4.1
-python-dateutil==2.8.2
-python-jose==3.2.0
-python-slugify==4.0.1
-pytz==2020.1
-pytzdata==2020.1
-PyYAML==5.4
-requests==2.25.1
-responses==0.12.0
-rsa==4.7
-s3transfer==0.4.2
-six==1.15.0
-sortedcontainers==2.4.0
-tabulate==0.8.9
-tblib==1.7.0
-text-unidecode==1.3
-toml==0.10.1
-toolz==0.11.2
-tornado==6.1
-tqdm==4.49.0
-troposphere==3.1.1
-urllib3==1.26.6
-websocket-client==1.2.1
-Werkzeug==2.2.3
-wsgi-request-logger==0.4.6
-xmltodict==0.12.0
-zappa==0.56.1
-zict==2.0.0
-zipp==3.4.0
+boto3==1.26.*
+Cerberus==1.3.*
+cryptography==39.0.*
+Flask==2.3.*
+Flask-Cognito==1.18.*
+Flask-Cors==3.0.*
+moto==4.1.*
+pytest==6.*
+requests==2.30.*
+zappa==0.56.*
```

Comment:
i settled on this shortened list of packages by grabbing the names of the packages we import directly in our own Python code. We do import `werkzeug` but since it's a dependency of `flask` and `zappa` so we don't need our pin.